### ISSUE TITLE
Settings editor Experimental and Preview indicators should be announced in "Screen Reader" mode

### ISSUE THREAD

**Issue by bhavyaus** [microsoft/vscode#231973]
Testing #231887

1. View any of the experimental settings in the Settings Editor
2. Turn on `Screen Reader Mode` and enable an accessibility viewer.
3. The `Preview` mode is not announced in the accessibility viewer unless the `Preview` tag is explicitly selected.

**Comment by rzhao271** [comment]
I see "Preview" show up in the NVDA Speech Viewer on Windows. 
1. Is this issue happening on the latest Insiders?
2. If yes to 1, which OS and screenreader are you using?
![Preview being read in NVDA Speech Viewer](https://github.com/user-attachments/assets/0e4fcdd1-6a9a-41f5-b777-5ae3430bfbd8)
